### DISCUSSION TITLE
nas-hardwares

### DISCUSSION THREAD

**Discussion by giscus** [bowencool/blog#79] [Announcements]
# nas-hardwares

我目前的 NAS 是由一台 8 年前的旧电脑改造而成的。现在打算更换硬件，参考了一些别人的方案，并调研了比较流行的 NAS 硬件。在此记录一下，也供有需要的人参考。

https://blog.bowen.cool/zh/posts/nas-hardwares

**Comment by meyergin** [comment]
B660M 停产不好买了吧

**Reply by bowencool** [reply]
确实是，我是在淘宝买的准新，带质保还可以
